### PR TITLE
Add Denis' old unmerged changes into main repo

### PR DESCRIPTION
**Pull request by aanil**
A new table `RoutingAction` was added in 2017 which was not merged with the main repo.

### CODE REVIEW

File: version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__=1.0.0
+__version__=1.0.1
```
Comment (alneberg) [inline version.py]: I think we should go for 1.1.0

**Review by alneberg — APPROVED** [review]
Looks good! Just bump the version a bit more.

File: version.py (hunk 1 above)
Comment (aanil) [inline version.py]:
```suggestion
__version__=1.1.0
```

Comment (aanil) [reply version.py]: Done!

**Pull request merged**